packages/preview/numblex/0.1.0/typst.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "numblex"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["ParaN3xus@https://github.com/ParaN3xus", "sjfhsjfh@https://github.com/sjfhsjfh"]
```

Comment:
I think this format is not valid. You can either use:
- just a name: `ParaN3xus`
- a name and an email address: `ParaN3xus <you@example.org>`
- a name and a homepage URL: `ParaN3xus <https://example.org>`
- a name and a GitHub username: `ParaN3xus <@ParaN3xus>` (which is probably the one you are looking for in that case)